src/SfxWeb/src/app/views/node/events/events.component.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    <app-event-store [listEventStoreData]="listEventStoreData" [optionsConfig]="optionsConfig"></app-event-store>
+    <app-event-store [listEventStoreData]="listEventStoreData" [optionsConfig]="optionsConfig" [isShown]="isShown"></app-event-store>
```

Comment:
is this a new input for the event store?

Reply:
yeah it is for an ngif for the table

Reply:
this should have a better variable name, something that is more descriptive of what its hiding. Additionally, it seems a bit hard to work with if we remove the list. It might be better if we put the events list inside a collapse container and had an Input for whether or not it should start open.